common/app/conf/switches/MonitoringSwitches.scala

```diff
@@ -31,8 +31,8 @@ trait MonitoringSwitches {
     "google-analytics",
```

Comment:
I think I'll leave it for a little longer, just in case the change to send 100% of requests to GA has a disastrous effect on site performance.